### DISCUSSION TITLE
Some torrents start fine, others can't find peers

### DISCUSSION THREAD

**Discussion by maltschuld** [haugene/docker-transmission-openvpn#2047] [Q&A] [answered]
I had this working perfectly for a while, even after upgrading to DSM 7 on my Synology NAS. The last time I added some torrents was about a month ago. I went to add some magnets today, and found that they were stuck at Downloading from 0 of 0 peers/ Magnetizing transfer- receiving metadata. There should be peers available for these. I tried a different tracker and found that torrents from TPB worked fine, but rarbg and 1337x would stay stuck in the magnetizing transfer state. 

ps I'm sort of a dummy and I've managed to get this working from Frankensteining many different guides together. Don't rule out some dumb mistake on my part. I don't know what to make of the logs. Any suggestions?

logs:
,curl: (7) Failed to connect to 10.37.112.1 port 19999: Connection refused
,Sun Oct 31 19:12:02  2021: bindPort error
,
,the has been a fatal_error
,60 day port reservation reached
,Getting a new one
,curl: (7) Failed to connect to 10.37.112.1 port 19999: Connection refused
,Sun Oct 31 19:12:02  2021: getSignature error
,
,the has been a fatal_error
,curl: (7) Failed to connect to 10.37.112.1 port 19999: Connection refused
,Sun Oct 31 19:12:03  2021: bindPort error
,
,the has been a fatal_error
,transmission auth not required
,waiting for transmission to become responsive
,transmission became responsive
,  31    n/a       None  Unknown      0.0     0.0   None  Idle         South.of.Heaven.2021.720p.WEBRip.800MB.x264-GalaxyRG
,Sum:          20.84 GB              22.0     0.0
,setting transmission port to 
,localhost:9091/transmission/rpc/ responded: "success"
,Checking port...
,Error: portTested: http error 400: Bad Request

env:
PATH	/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
OPENVPN_USERNAME	xxxxxxxxxxx
OPENVPN_PASSWORD	xxxxxxxxxxx
OPENVPN_PROVIDER	pia
GLOBAL_APPLY_PERMISSIONS	true
TRANSMISSION_HOME	/data/transmission-home
TRANSMISSION_RPC_PORT	9091
TRANSMISSION_DOWNLOAD_DIR	/Data/Downloads/Torrents/Movies
TRANSMISSION_INCOMPLETE_DIR	/Data/Downloads/Torrents/InProgress
TRANSMISSION_WATCH_DIR	/Data/Downloads/Torrents/ToFetch
CREATE_TUN_DEVICE	true
ENABLE_UFW	false
UFW_ALLOW_GW_NET	false
UFW_EXTRA_PORTS	
UFW_DISABLE_IPTABLES_REJECT	false
PUID	xxxx
PGID	xxxx
PEER_DNS	true
PEER_DNS_PIN_ROUTES	true
DROP_DEFAULT_ROUTE	
WEBPROXY_ENABLED	false
WEBPROXY_PORT	8888
WEBPROXY_USERNAME	
WEBPROXY_PASSWORD	
LOG_TO_STDOUT	false
HEALTH_CHECK_HOST	google.com
SELFHEAL	false
REVISION	3d97cd5302985c1a710f46ab0c311f721f224fc6
PIA_OPENVPN_CONFIG_BUNDLE	openvpn
TRANSMISSION_RPC_WHITELIST	192.168.1.148
TRANSMISSION_RPC_WHITELIST_ENABLED	false
TRANSMISSION_UPLOAD_SLOTS_PER_TORRENT	5
TRANSMISSION_UPLOAD_LIMIT_ENABLED	true
TRANSMISSION_UPLOAD_LIMIT	10
TRANSMISSION_START_ADDED_TORRENTS	true
TRANSMISSION_SPEED_LIMIT_UP_ENABLED	true
TRANSMISSION_SPEED_LIMIT_UP	25
TRANSMISSION_RATIO_LIMIT_ENABLED	true
TRANSMISSION_RATIO_LIMIT	2
TRANSMISSION_QUEUE_STALLED_MINUTES	90
TRANSMISSION_QUEUE_STALLED_ENABLED	true
TRANSMISSION_MAX_PEERS_GLOBAL	200
TRANSMISSION_IDLE_SEEDING_LIMIT_ENABLED	true
TRANSMISSION_IDLE_SEEDING_LIMIT	30
TRANSMISSION_DOWNLOAD_QUEUE_SIZE	12
TRANSMISSION_DOWNLOAD_QUEUE_ENABLED	true
TRANSMISSION_DOWNLOAD_LIMIT_ENABLED	false
TRANSMISSION_DOWNLOAD_LIMIT	100
TRANSMISSION_ALT_SPEED_ENABLED	false
TRANSMISSION_ALT_SPEED_DOWN	50
OVERRIDE_DNS_2	8.8.4.4
OVERRIDE_DNS_1	8.8.8.8
TRANSMISSION_RPC_PASSWORD	xxxxxxxx
TRANSMISSION_RPC_USERNAME	xxxxxxxx
TZ	us-central
OPENVPN_CONFIG	us_chicago,us_denver
LOCAL_NETWORK	192.168.1.0/24
OPENVPN_OPTS	--inactive 3600 --ping 10 --ping-exit 60 --mute-replay-warnings
TRANSMISSION_UMASK	002
TRANSMISSION_WATCH_DIR_ENABLED	true

**Comment by pkishino** [comment, marked as answer]
If you look at the logs it tells you the problem.. it cannot bind to the port forwarding port for the vpn server.
Have you tried restarting the container ?

**Reply by maltschuld** [reply]
Yes, I've tried restarting container and NAS several times. I don't recall having to setup any port forwarding previously. Any suggestions on how to go about fixing this? I'm using PIA.

**Reply by pkishino** [reply]
1)try different servers
2) delete container and try from scratch and use default server (ie, don't set OPENVPN_CONFIG)
3) check with PIA if they have changed stuff

**Reply by maltschuld** [reply]
Thanks. I had the same idea but just now had a chance to test it out. I changed openvpn_config to ca_toronto and that seemed to work. I still want to figure out why but atleast things are seeding as expected now. thanks!

**Reply by pkishino** [reply]
well, in that case it is an issue with the server chosen.. PIA support might be able to help you
